dblookupext/historyRepository.go

```diff
@@ -416,15 +417,15 @@ func (hr *historyRepository) OnNotarizedBlocks(shardID uint32, headers []data.He
 
 		log.Trace("onNotarizedBlocks():", "shardID", shardID, "nonce", headerHandler.GetNonce(), "headerHash", headerHash, "type", fmt.Sprintf("%T", headerHandler))
 
-		metaBlock, isMetaBlock := headerHandler.(*block.MetaBlock)
+		metaBlock, isMetaBlock := headerHandler.(data.MetaHeaderHandler)
 		if isMetaBlock {
-			for _, miniBlock := range metaBlock.MiniBlockHeaders {
+			for _, miniBlock := range metaBlock.GetMiniBlockHeaderHandlers() {
```

Comment:
shouldn't this get them from execution results if headerV3? common.GetMiniBlockHeadersFromExecResult